packages/amplication-prisma-db/prisma/schema.prisma

```diff
@@ -616,5 +616,4 @@ enum CodeGeneratorVersionStrategy {
 enum PreviewAccountType {
   None
   BreakingTheMonolith
-  Auth0Signup
```

Comment:
This was already merged to master and deployed to production. Are we good with removing it?

Reply:
forgot to push the manual data migration script. Good catch!

Reply:
Pushed [6108abc](https://github.com/amplication/amplication/pull/7916/commits/6108abcd06665e4005477a4628f9e8cdfeb6fa52)